Components/mos_ver.py

```
@@ -0,0 +1,43 @@
+    path = Path(__file__).resolve().parent.parent
+    return path.joinpath("build_version.txt").read_text().strip()
+
+
+#print("Version:", read_version())
+#print("Build ID:", read_build_id())
+
+class MosSettings(MosWindow):
+    def __init__(self, configl: dict):
+        self.logo = pygame.Surface()
+        config ={
+            'title': f"System Settings",
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)